python/cudf/cudf/core/frame.py

```diff
@@ -238,6 +239,68 @@ def _quantiles(
         result._copy_categories(self)
         return result
 
+    def repeat(self, repeats, axis=1):
```

Comment:
This needs to be `None` to be compatible with the Pandas signature.
```suggestion
    def repeat(self, repeats, axis=None):
```